public/index.html

```diff
@@ -4,7 +4,9 @@
     <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1,minimal-ui" />
     <link rel="shortcut icon" type="image/x-icon" href="https://decentraland.org/images/icons/favicon-32x32.png" />
-    <link rel="preload" href="https://decentraland.org/images/background@blur.jpg" as="image" />
+    <link rel="preload" href="./images/background-v2@blur.jpg" as="image" />
```

Comment:
Please use %PUBLIC_URL% everywhere in the HTML to make it work with the static CDN

```suggestion
    <link rel="preload" href="%PUBLIC_URL%/images/background-v2@blur.jpg" as="image" />
```